Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM neighborhoods/nginx-auth-ldap:1.0
+FROM neighborhoods/nginx-auth-ldap:1.2.0
```

Comment:
Adds the ca-certificates package in 1.1.0 (in which this repo already does programmatically in docker/build.sh). Putting it in the base feels like the right thing to. 
1.2.0 Upgrades both the nginx and alpine images used in order to compile nginx with openssl instead of libressl. Note libressl was a dependency of open-ldap-dev which switch to openssl in later versions. Nginx-ldap-auth requires openssl, not libressl.
https://github.com/neighborhoods/docker-nginx-auth-ldap/releases